Migration script/openIMIS migration latest.sql

```diff
@@ -150,6 +150,23 @@ BEGIN
 END
 GO
 
+DROP FUNCTION IF EXISTS [dbo].[udfDefaultLanguageCode] 
```

Comment:
Move SP in SP section after structural changes.

Reply:
You can't, if you do, you won't be able to create table tblFamlySMS